.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @adyen-examples/developer-relations
```

Comment:
Github mentions that the CODEOWNERS file contains error, please have a look 👍

Reply:
Added DevRel as mantainer